src/playbooks/deploy/deploy.yaml

```diff
@@ -32,6 +32,9 @@
     foreman_client_certificate: "{{ client_certificate }}"
     foreman_oauth_consumer_key: abcdefghijklmnopqrstuvwxyz123456
     foreman_oauth_consumer_secret: abcdefghijklmnopqrstuvwxyz123456
+    foreman_listen_stream: /run/httpd.foreman.sock
+    foreman_url: "https://{{ ansible_fqdn }}"
+    httpd_foreman_backend: "unix://{{ foreman_listen_stream }}|http://%{HTTP_HOST}/"
```

Comment:
I'm surprised all three of these are not the role defaults.

Reply:
Because they're connected: my thought was that the roles should be dependent and only on the playbook level you can relate them.